src/Aspire.Hosting/IInteractionService.cs

```diff
@@ -194,24 +194,63 @@ await _options.UpdateInputCallback(new UpdateInputContext
     internal Action<InteractionInput>? OnDataRefresh { get; set; }
 }
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+/// <summary>
+/// Represents configuration options for dynamically loading input data.
+/// </summary>
+/// <remarks>
+/// Use this class to specify how and when dynamic input data should be loaded. This type is intended for advanced
+/// scenarios where input loading behavior must be customized.
+/// </remarks>
 [Experimental(InteractionService.DiagnosticId, UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
-public sealed class DynamicInputOptions
+public sealed class DynamicInputLoading
```

Comment:
Feels like this is missing a noun

Reply:
Can this and `LoadInputContext` be named similar?

* LoadInputContext / LoadInputOptions
* DynamicLoadingContext / DynamicLoadingOptions
* etc